File: jimmer-ddd-java-sample/domain/src/main/java/org/morecup/jimmerddd/java/sample/domain/order/Gift.java
package org.morecup.jimmerddd.java.sample.domain.order;

import org.babyfish.jimmer.sql.Entity;
import org.babyfish.jimmer.sql.ManyToMany;
import org.morecup.jimmerddd.java.sample.domain.base.BaseEntity;
import java.util.List;

@Entity
public interface Gift extends BaseEntity {
    
    String giftName();

    @ManyToMany
    List<Order> orderList();
}

File: jimmer-ddd-kotlin-sample/domain/src/main/kotlin/org/morecup/jimmerddd/kotlin/sample/domain/order/Gift.kt
package org.morecup.jimmerddd.kotlin.sample.domain.order

import org.babyfish.jimmer.sql.Entity
import org.babyfish.jimmer.sql.ManyToMany
import org.morecup.jimmerddd.kotlin.sample.domain.base.BaseEntity

@Entity
interface Gift: BaseEntity {

    val giftName: String


    @ManyToMany
    val orderList:List<Order>
}